frontend/src/lib/lemon-ui/LemonTag/LemonTag.tsx

```diff
@@ -30,7 +30,7 @@ export interface LemonTagProps extends React.HTMLAttributes<HTMLDivElement> {
     popover?: LemonButtonDropdown
 }
 
-export function LemonTag({
+export const LemonTag = ({
```

Comment:
React function components should be actual `function`s, because this way they get `displayName` automatically. With `const`s you have to set `LemonTag.displayName = 'LemonTag'` manually